Project/ProjectSettings/UnityConnectSettings.asset

```diff
@@ -4,7 +4,7 @@
 UnityConnectSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 1
-  m_Enabled: 1
+  m_Enabled: 0
```

Comment:
Automatically set during player build. Might as well commit it now so we don't have to keep undoing it.